README.md

```diff
@@ -313,11 +313,13 @@ import {ValidateNested} from "class-validator";
 export class Post {
 
     @ValidateNested()
-    user: User;
+    user: User = new User();
 
 }
 ```
 
+Please note that nested object **must** be created using `new` keyword, otherwise `@ValidateNested` won't work.
```

Comment:
I think this note is a little bit misleading. `new` keyword is not the only way how to create instance of class (e.g. `class-transformer`). `class-validator` use `object.constructor` as target for validation so we should change this note to something like:

Please note that nested object **must** be instance of class, otherwise `@ValidateNested` won't know what class is target of validation. Check also [Validating plain objects](https://github.com/typestack/class-validator#validating-plain-objects).

Or something like that.

Reply:
Thanks @vlapo for reviewing. Your option sounds much better. I update PR, please have a look